Task :persistSentryProguardUuidsForReleaseArm64-v8a-release FAILED (AS:3.3 ABP:3.3.0 Gradle:4.10.1 sentry-android-gradle-plugin:1.7.16)

Project was working fine with sentry plugin before upgrade to AS 3.3 and the matching dev tools.

Now I have more build variants than just release/debug which I had before.

When I build a release version of the APK I get "Task :persistSentryProguardUuidsForReleaseArm64-v8a-release FAILED". The plugin is trying to run command:

```
"[user]\\AppData\\Local\\Temp\\.sentry-cli8452420388174734644.exe" "upload-proguard" "--android-manifest" "[project_root]\\build\\intermediates\\merged_manifests\\release\\AndroidManifest.xml" "--write-properties" "[project_root]\\build\\intermediates\\merged_assets\\release\\mergeReleaseAssets\\out/sentry-debug-meta.properties" "[project_root]\\build\\outputs\\mapping\\release\\mapping.txt"
```

but there is no ``"[project_root]\\build\\intermediates\\merged_manifests\\release\\AndroidManifest.xml"``, instead there is ``"[project_root]\\build\\intermediates\\merged_manifests\\release\\arm64-v8a\\AndroidManifest.xml"``. The task fails with "error: The system cannot find the file specified. (os error 2)".

I am running the build on Windows 10.

Commented out the "splits" section and tried to build without it. Manifest was created as expected and cli started fine, but then got error from server:

```
> Task :persistSentryProguardUuidsForReleaseRelease
  INFO    2019-01-21 21:03:01.053459300 +02:00 Loaded config from [user]\.sentryclirc
  DEBUG   2019-01-21 21:03:01.053459300 +02:00 sentry-cli version: 1.37.4, platform: "windows", architecture: "i686"
  INFO    2019-01-21 21:03:01.053459300 +02:00 sentry-cli was invoked with the following command line: "[user]\\AppData\\Local\\Temp\\.sentry-cli8081640012938934376.exe" "upload-proguard" "--android-manifest" "[project root]\\build\\intermediates\\merged_manifests\\release\\AndroidManifest.xml" "--write-properties" "[project root]\\build\\intermediates\\merged_assets\\release\\mergeReleaseAssets\\out/sentry-debug-meta.properties" "[project root]\\build\\outputs\\mapping\\release\\mapping.txt"
> compressing mappings
> uploading mappings
  DEBUG   2019-01-21 21:03:04.388949800 +02:00 request POST https://sentry.io/api/0/projects/x/y/files/dsyms/
  DEBUG   2019-01-21 21:03:04.388949800 +02:00 using token authentication
  DEBUG   2019-01-21 21:03:04.388949800 +02:00 sending form data
  DEBUG   2019-01-21 21:03:04.388949800 +02:00 retry number 0, max retries: 0
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > POST /api/0/projects/x/y/files/dsyms/ HTTP/1.1
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Host: sentry.io
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Accept: */*
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Connection: TE
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > TE: gzip
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > User-Agent: sentry-cli/1.37.4
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Authorization: Bearer 679cd84f***
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Content-Length: 624181
  DEBUG   2019-01-21 21:03:04.940606500 +02:00 > Content-Type: multipart/form-data; boundary=------------------------0036d46a2c3a52df
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < HTTP/1.1 400 BAD REQUEST
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Server: nginx
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Date: Mon, 21 Jan 2019 19:03:10 GMT
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Content-Type: application/json
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Content-Length: 35
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Connection: keep-alive
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < X-XSS-Protection: 1; mode=block
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Content-Language: en
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < X-Content-Type-Options: nosniff
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Vary: Accept-Language, Cookie
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Allow: GET, POST, DELETE, HEAD, OPTIONS
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < X-Frame-Options: deny
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < X-Served-By: web-b378b859
  DEBUG   2019-01-21 21:03:10.765475800 +02:00 < Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
  DEBUG   2019-01-21 21:03:10.766473200 +02:00 response status: 400
  DEBUG   2019-01-21 21:03:10.766473200 +02:00 body: {"detail": "Missing uploaded file"}
  DEBUG   2019-01-21 21:03:10.766473200 +02:00 error: running update nagger
  DEBUG   2019-01-21 21:03:10.766473200 +02:00 skipping update nagger because session is not attended
error: API request failed
  caused by: sentry reported an error: Missing uploaded file (http status: 400)
  DEBUG   2019-01-21 21:03:10.767469 +02:00 client close; no transport to shut down  (from sentry)

> Task :persistSentryProguardUuidsForReleaseRelease FAILED

FAILURE: Build failed with an exception.
```

mapping file is 10MB.

Seems like the plugin is not working well with ABI split projects using the new build tools.

Seems like there is another issue with uploads.

--- a/sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy
+++ b/sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy
@@ -168,7 +168,7 @@ class SentryPlugin implements Plugin<Project> {
 
                     if (manifestPath == null) {
                         def dir = findAndroidManifestFileDir(variantOutput)
-                        manifestPath = new File(dir, "AndroidManifest.xml")
+                        manifestPath = new File(new File(dir, variantOutput.dirName), "AndroidManifest.xml")
                     }
 
                     def mappingFile = variant.getMappingFile()

